mapie/tests/test_mondrian.py

```diff
@@ -0,0 +1,434 @@
+from copy import deepcopy
+
+import numpy as np
+import pytest
+from sklearn.base import clone
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.datasets import (
+    make_classification,
+    make_multilabel_classification,
+    make_regression
+)
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.model_selection import ShuffleSplit
+
+from mapie.calibration import MapieCalibrator
+from mapie.classification import MapieClassifier
+from mapie.conformity_scores import (
+    AbsoluteConformityScore,
+    APSConformityScore,
+    GammaConformityScore,
+    LACConformityScore,
+    TopKConformityScore,
+    RAPSConformityScore,
+    ResidualNormalisedScore
+)
+from mapie.mondrian import MondrianCP
+from mapie.multi_label_classification import MapieMultiLabelClassifier
+from mapie.regression import (
+    MapieQuantileRegressor,
+    MapieRegressor,
+    MapieTimeSeriesRegressor
+)
+
+VALID_MAPIE_ESTIMATORS = {
+    "classif_score": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"method": "score"}
+    },
+    "classif_lac": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"method": "lac"}
+    },
+    "classif_aps": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"method": "aps"}
+    },
+    "classif_cumulated_score": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"method": "cumulated_score"}
+    },
+    "classif_topk": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"method": "top_k"}
+    },
+    "classif_lac_conformity": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"conformity_score": LACConformityScore()}
+    },
+    "classif_aps_conformity": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"conformity_score": APSConformityScore()}
+    },
+    "classif_topk_conformity": {
+        "estimator": MapieClassifier,
+        "task": "classification",
+        "kwargs": {"conformity_score": TopKConformityScore()}
+    },
+    "regression_absolute_conformity": {
+        "estimator": MapieRegressor,
+        "task": "regression",
+        "kwargs": {"conformity_score": AbsoluteConformityScore()}
+    },
+    "regression_none": {
+        "estimator": MapieRegressor,
+        "task": "regression",
+        "kwargs": {"conformity_score": None}
+    },
```

Comment:
```suggestion
    "regression_none": {
        "estimator": MapieRegressor,
        "task": "regression",
        "kwargs": {"conformity_score": AbsoluteConformityScore()}
    },
```

Reply:
To avoid confusion when reading

Reply:
I want to test both with default value and Absolute is done above